### DISCUSSION TITLE
.env file Error

### DISCUSSION THREAD

**Discussion by akhilparakka** [smartcontractkit/full-blockchain-solidity-course-py#1591] [Q&A] [answered]
.env file shows many errors. Please Help.
![Screenshot from 2022-07-21 12-18-05](https://user-images.githubusercontent.com/77268333/180148380-2832b8e7-0d01-43a0-82c1-8ec9d31d2035.png)

**Comment by cromewar** [comment, marked as answer]
Hello @Akhil-franko 
Install the .env extension on the store and that should solve your problem

**Reply by akhilparakka** [reply]
Thank You.

**Reply by dankorea** [reply]
installed .env extension but not solved
